### PR TITLE
Fix/tablet card styles

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
**Raising PR on behalf of Priyanka**

#### Changes proposed in this Pull Request:

- [X] Fix the border radius of the card
- [X] Fix padding between header and instructions
- [X] Fix padding as highlighted in your second graphic
- [X] Fix the font size on the set password criteria
- [X] Update the show/hide icon to use the outlined style
- [X] Update card background colour "surface container"

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/components/WorkflowCard/WorkflowCard.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const makeStyles = ({
         tablet: {
             height: MAX_CARD_HEIGHT,
             width: MAX_CARD_WIDTH,
-            borderRadius: theme.roundness * 2,
+            borderRadius: theme.roundness * 10,
```
Comment (manojleaton) [inline login-workflow/src/components/WorkflowCard/WorkflowCard.tsx:36]: I think we need to add 24px of border radius to the card. `theme.roundness` is 4px so it should be multiple of 6.
```suggestion
            borderRadius: theme.roundness * 6,
```

**Comment by manojleaton** [comment]
Vertical padding is not matching with the design mentioned in the issue, Please add both Vertical and Horizontal padding of 24px.
Screenshot 2024-03-26 at 3 23 20 PM

**Comment by JeffGreiner-eaton** [comment]
The vertical on iPad (24) and phone (16) looks good. @manojleaton you mentioned horizontal adjustment. Should this all be 24 vertical & horizontal on iPad? and all 16 on phone?

image

File: login-workflow/src/components/SetPassword/PasswordTextField.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ export const PasswordTextField: React.FC<TextInputProps> = forwardRef((props, re
                     testID="toggle-button"
                     icon={showPassword ? 'eye' : 'eye-off'}
                     onPress={() => setShowPassword(!showPassword)}
+                    mode="outlined"
```
Comment (ektaghag-eaton) [inline login-workflow/src/components/SetPassword/PasswordTextField.tsx:28]: Hi @priyankakmEaton , please remove this mode.

**Review by surajeaton — APPROVED** [review]

**Pull request merged**